src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -127,27 +125,25 @@ describe('ATI Analytics Container', () => {
 
       renderer.create(
         <ContextWrap platform="amp" pageType="frontPage">
-          <ATIAnalytics data={mockData} />
+          <ATIAnalytics data={articleDataNews} />
         </ContextWrap>,
       );
 
       expect(mockAmp).toHaveBeenCalledTimes(1);
-      expect(mockAmp).toHaveBeenCalledWith(
-        {
-          pageviewParams: mockAtiQueryParams,
-        },
-        mockData,
-      );
+      expect(mockAmp.mock.calls[0][0]).toEqual({
+        pageviewParams: mockAtiQueryParams,
+      });
+
       expect(mockFrontPageAtiParams).toHaveBeenCalledTimes(1);
-      expect(mockFrontPageAtiParams).toHaveBeenCalledWith(mockData);
+      expect(mockFrontPageAtiParams).toHaveBeenCalledWith(articleDataNews);
     });
   });
 
   describe('pageType neither article nor frontPage', () => {
     isNull(
       'should render null',
-      <ContextWrap platform="canonical" pageType="randomvalue">
-        <ATIAnalytics data={mockData} />
+      <ContextWrap platform="canonical" pageType="media">
```

Comment:
pageType `media` is in development atm which will break this, why did this need changing from `'randomvalue'`?

Reply:
```
Warning: Failed prop type: Invalid prop `pageType` of value `randomvalue` supplied to `RequestContextProvider`, expected one of ["article","frontPage","media"].
  in RequestContextProvider (created by ContextWrap)
  in ContextWrap
```

We are trying to fix this React.warning above and media is the only valid prop we have left for this test.

Reply:
pageType expects "article", "frontPage" or "media". The test says that the pageType can neither be article nor frontPage therefore I chose media.

````
   Warning: Failed prop type: Invalid prop `pageType` of value `randomvalue` supplied to `RequestContextProvider`, expected one of ["article","frontPage","media"].
          in RequestContextProvider (created by ContextWrap)
          in ContextWrap
````

Reply:
after speaking irl:

We agreed that it may cause issues for the media pod and this change should be undone.
Instead we would supress this warning once this issue is complete: https://github.com/bbc/simorgh/issues/3107

Reply:
discussed offline with @thekp - we agreed to skip this test until we can choose to skip prop type checks in specific tests to avoid confusion with others